photon-thinclient/thinclient.html

```diff
@@ -0,0 +1,296 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>ThinClient</title>
+
+    <style>
+        * {
+            margin: 0;
+            padding: 0;
+        }
+        .imgbox {
```

Comment:
I think the thinclient folder needs to be removed...>?

Reply:
Just adding it to gitignore or something else?